kubernetes/templates/exareme2-globalnode.yaml

```diff
@@ -1,55 +1,77 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: exareme2-globalworker
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: exareme2-worker
+    nodeType: globalworker
+spec:
+  clusterIP: None
+  selector:
+    app: exareme2-worker
+    nodeType: globalworker
+  ports:
+    - name: rabbitmq
+      port: 5672
+    - name: monetdb
+      port: 50000
+---
 apiVersion: apps/v1
-kind: Deployment
+kind: StatefulSet
 metadata:
-  name: exareme2-globalworker-deployment
+  name: exareme2-globalworker
   namespace: {{ .Values.namespace }}
   labels:
     app: exareme2-worker
     nodeType: globalworker
 spec:
+  serviceName: exareme2-globalworker
   replicas: 1
   selector:
     matchLabels:
       app: exareme2-worker
+      nodeType: globalworker
   template:
     metadata:
       labels:
         app: exareme2-worker
         nodeType: globalworker
     spec:
+      {{ if not .Values.managed_cluster }}
       nodeSelector:
         master: "true"
+      {{ end }}
+
+        # Choose storage backend: hostPath or dynamically provisioned CephFS PVCs
+      {{ if not .Values.managed_cluster }}
       volumes:
         - name: db-data
           hostPath:
-            path: {{ printf "%s/db" .Values.db.globalworker_location }}
+            path: {{ printf "%s/db" .Values.storage.hostPath.db.globalworker }}
         - name: csv-data
           hostPath:
-            path: {{ printf "%s/csvs" .Values.db.globalworker_location }}
+            path: {{ printf "%s/csvs" .Values.storage.hostPath.db.globalworker }}
         - name: credentials
           hostPath:
-            path: {{ printf "%s/credentials" .Values.db.globalworker_location }}
-
+            path: {{ printf "%s/credentials" .Values.storage.hostPath.db.globalworker }}
+      {{ end }}
       containers:
       - name: monetdb
         image: {{ .Values.exareme2_images.repository }}/exareme2_db:{{ .Values.exareme2_images.version }}
-        # LIMITS REMOVED DUE TO MONETDB NOT SEEING THEM PROPERLY
-        # resources:
-        #  limits:
-        #    memory: "{{ .Values.db.max_memory }}Mi"
         env:
         - name: LOG_LEVEL
           value: {{ .Values.log_level }}
         - name: MONETDB_NCLIENTS
-          value: {{ mul .Values.max_concurrent_experiments .Values.localnodes | quote }}
-        - name: MAX_MEMORY   # Value in bytes
-          value: {{ mul .Values.db.max_memory 1048576 | quote }} # Conversion to bytes
-        - name: SOFT_RESTART_MEMORY_LIMIT   # Value in megabytes
-          value: {{ div  (mul .Values.db.max_memory .Values.db.percentage_soft_memory_limit) 100 | quote }}
-        - name: HARD_RESTART_MEMORY_LIMIT   # Value in megabytes
-          value: {{ div  (mul .Values.db.max_memory .Values.db.percentage_hard_memory_limit) 100 | quote }}
+          value: {{ mul .Values.max_concurrent_experiments 1 | quote }}
```

Comment:
Why did you change the multiplication here? If there is a reason you can just remove the multiplication.